hw0/README.rst

```diff
@@ -44,6 +44,6 @@ run the following commands:
 .. code-block:: bash
 
   cd nsdhw_21au/hw0/username
-  ../validate.py | grep "GET POINT"
+  ../validate.sh | grep "GET POINT"
```

Comment:
Please do not include changes of other branch.  It seems that this PR did not branch from `hw1`.

Reply:
Oh, I have accidentally branched from master.
I have made another PR #21.

Reply:
You can close this one to avoid confusing the grader.  I am closing it for you.  Don't forget to mark your PR URL at E3.